src/display.jl

```diff
@@ -1,379 +1,405 @@
 mutable struct DisplayParameters
     format::Symbol
     decorations::Bool
-    sigfigs::Int
+    sigdigits::Int
 end
 
-function Base.show(io::IO, params::DisplayParameters)
-    println(io, "Display parameters:")
-    println(io, "- format: $(params.format)")
-    println(io, "- decorations: $(params.decorations)")
-    print(io, "- significant figures: $(params.sigfigs)")
+function Base.show(io::IO, ::MIME"text/plain", params::DisplayParameters)
+    println(io, "Display options:")
+    println(io, "  - format: ", params.format)
+    println(io, "  - decorations: ", params.decorations)
+    print(io, "  - significant digits: ", params.sigdigits)
 end
 
-const display_params = DisplayParameters(:standard, false, 6)
-
-const display_options = (:standard, :full, :midpoint)
+const display_params = DisplayParameters(:standard, false, 6)  # Default
```

Comment:
What about displaying by default the decoration? If you care about `DecoratedIntervals`, you want to see it!

Reply:
That is a good idea, I think that would be a nice improvement 🙂.